src/complex.rs

```diff
@@ -64,7 +66,7 @@ fn rearrange<T>(data: &mut [Complex<T>], n: usize) {
 #[inline(always)]
 fn scale<T>(data: &mut [Complex<T>], n: usize) where T: Float {
     let factor = T::from(n).unwrap().recip();
-    for i in 0..n {
```

Comment:
Yes i see but do explicit indexing via [..] don't give you bounds checking penalty?